### ISSUE TITLE
Phpcbf path is not defined

### ISSUE THREAD

**Issue by Aporie** [Glavin001/atom-beautify#2140]
# Description

Was trying to setup drupal standard for phpcbf so I've made a new global install of it with composer. After entering the new path in the executable, running beautify result in "error path is not defined".

# Input Before Beautification

Not relevant

# Expected Output

Not relevant

# Actual Output

Not relevant

# Steps to Reproduce

```
composer global require drupal/coder
composer global require dealerdirect/phpcodesniffer-composer-installer
```
Setting up path to phpcbf in atom beautify settings
Optionally setting up phpcbf standard to Drupal,DrupalPractice (neither works with PEAR).

# Debug

Here is a link to the `debug.md` Gist: https://gist.github.com/Aporie/2e43838a1a2d1af321dd44487862e97c

# Checklist

I have:
- [ ] Tried uninstalling and reinstalling Atom Beautify to ensure it installed properly
- [X] Reloaded (or restarted) Atom to ensure it is not a caching issue
- [X] Searched through existing Atom Beautify Issues at https://github.com/Glavin001/atom-beautify/issues
so I know this is not a duplicate issue
- [X] Filled out the Input, Expected, and Actual sections above or have edited/removed them in a way that fully describes the issue.
- [X] Generated debugging information by executing `Atom Beautify: Help Debug Editor` command in Atom and added link for `debug.md` Gist to this issue

**Comment by szeck87** [comment]
@Aporie I see the specific issue with the path, that definitely needs to be fixed.  But it looks like you are installing it globally.  Can you try running `phpcs` and/or `phpcbf` from the terminal (just get the version or something) and see if it runs?  If so, you probably don't even need the paths there.

**Comment by Aporie** [comment]
Hi @szeck87,

No I haven't installed it globally (and I don't really want to) : 

```
$ phpcbf -i
bash: phpcbf: command not found...
Install package 'php-pear-PHP-CodeSniffer' to provide command 'phpcbf'? [N/y]
```
Whereas :
```
$ .config/composer/vendor/squizlabs/php_codesniffer/scripts/phpcbf -i
The installed coding standards are PEAR, PSR2, Squiz, MySource, Zend, PSR1, PHPCS, DrupalPractice and Drupal
```

**Comment by Aporie** [comment]
I've tested your PR and it did the trick :) thanks.

**Comment by szeck87** [comment]
Fixed with v0.32.5.
